api/dao/hierarchy.py

```diff
@@ -319,53 +319,59 @@ def dict_fileinfos(infos):
     return dict_infos
 
 
-def update_container_hierarchy(metadata, acquisition_id, level):
-    project = metadata.get('project')
-    session = metadata.get('session')
-    acquisition = metadata.get('acquisition')
+def update_container_hierarchy(metadata, cid, container_type):
```

Comment:
Could you add a docstring with a few sentences describing what this does? I'm not sure if I've ever understood what (exactly) this function does without reverse-engineering it each time I read it :)